api/Jobs/SyncAssignedCasesJob.cs

```diff
@@ -225,6 +220,38 @@ private async Task ProcessScheduledCases()
         this.Logger.LogInformation("Processed {Count} Scheduled Cases", scheduledData.Count);
     }
 
+    private async Task<ICollection<Case>> GetScheduledCases()
+    {
+        var apprReasonCodes = string.Join(",", CaseService.ContinuationReasonCodes);
+
+        // Retrieve the Scheduled Cases by invoking a lambda function to avoid API Gateway's timeout.
+        // The endpoint is expected to take a while to get the data specially in PROD.
+        var lambdaName = this.Configuration.GetValue<string>("AWS_GET_ASSIGNED_CASES_LAMBDA_NAME");
+        if (!string.IsNullOrWhiteSpace(lambdaName))
```

Comment:
Wade already commented on this, so no changes necessary, but this could be a good place for an interface to handle splitting the calls based on configuration.